### PR TITLE
PIM-5029: rework AjaxReference transformer

### PR DESCRIPTION
**Pull request by sumbobyboys**
| Q                 | A
| ----------------- | ---
| Specs             | N
| Behats            | N
| Blue CI           | N
| Changelog updated | N
| Review and 2 GTM  | N
| Micro Demo (PO)   | N
| Migration script  | NA
| Tech Doc          | N

### CODE REVIEW

File: features/Context/Page/Base/Grid.php [hunk 1]
```diff
@@ -296,19 +292,17 @@ public function changePageSize($num)
...
-
-                return true;
-            }
-
-            return false;
-        });
+            return $this->getGrid()
+                ->getParent()
+                ->getParent()
+                ->getParent()
```
Comment (pierallard) [inline features/Context/Page/Base/Grid.php]: Is there any `closest` as jQuery instead of this ?

**Comment by pierallard** [comment]
If there is no better solution than my comment, so GTM

**Comment by jjanvier** [comment]
gtm

**Comment by ClementGautier** [comment]
looks good to me, GTM

File: features/Context/Page/Base/Grid.php (hunk 1 above)
Comment (sumbobyboys) [inline features/Context/Page/Base/Grid.php]: nope :/

**Pull request merged**